docs/architecture/deep-dives/authentication/index.md

```diff
@@ -0,0 +1,325 @@
+# Authentication
+
+## Client Login Strategies
+
+Our authentication methods are defined via “strategies” in our client code. These strategies are:
+
+- Password Login Strategy: Authentication with an email address and master password.
+- Passwordless Login Strategy: Authentication with a
+  [Login with Device](https://bitwarden.com/help/log-in-with-device/) one-time access code.
+- API Login Strategy: Authentication with an API key and secret.
+- SSO Login Strategy: Authentication with an SSO IdP through SAML or OpenIDConnect.
+
+Regardless of the strategy, “logging in” to Bitwarden requires `POST`ing a request to the
+`/connect/token` endpoint on our Identity server. The contents of this request vary by several
+factors, but the most important is the login strategy.
+
+## Step 1: Client requests a token
```

Comment:
❓ Is this "step" wording necessary, or is its flow implied? When compared to other pages I think it can be dropped.

Reply:
I'm good with that.  It helped sequentialize it in my head when trying to write it all out, but it doesn't seem necessary now.